pyiceberg/catalog/sql.py

```diff
@@ -110,8 +114,14 @@ def __init__(self, name: str, **properties: str):
 
         if not (uri_prop := self.properties.get("uri")):
             raise NoSuchPropertyException("SQL connection URI is required")
-        echo = bool(self.properties.get("echo", False))
-        self.engine = create_engine(uri_prop, echo=echo)
+
+        echo: Union[str, bool] = "debug"
+        echo_str = str(self.properties.get("echo", DEFAULT_ECHO_VALUE)).lower()
+        if echo_str != "debug":
+            echo = strtobool(echo_str)
```

Comment:
nit: this logic is confusing to me. maybe something like 
```
echo_str = str(self.properties.get("echo", DEFAULT_ECHO_VALUE)).lower()
echo = strtobool(echo_str) if echo_str != "debug" else "debug"
```

Reply:
Agree, yours is easier to read.  👏🏻